databricks/resource_databricks_azure_adls_gen2_mount_test.go

```diff
@@ -39,7 +67,7 @@ func TestAccAzureAdlsGen2Mount_capture_error(t *testing.T) {
 }
 
 func testAccAzureAdlsGen2Mount_correctly_mounts() string {
-	clientID := os.Getenv("ARM_CLIENT_ID")
+	clientID := os.Getenv("DATABRICKS_AZURE_CLIENT_ID")
```

Comment:
What was the reason for changing this?

Reply:
hmmmmmm, i think this was when i was struggling to get the tests running as the .env wasn't applying and clientID was being set to null causing tests to fail.

I since figured out the issue there so can probably roll this back. The actual values of `ARM_CLIENT_ID` and `DATABRICKS_AZURE_CLIENT_ID` are the same though.

Will revert this change :)